go/consensus/tendermint/apps/staking/signing_rewards.go

```diff
@@ -0,0 +1,84 @@
+package staking
+
+import (
+	"bytes"
+	"fmt"
+	"math"
+	"sort"
+
+	"github.com/oasislabs/oasis-core/go/common/crypto/signature"
+	"github.com/oasislabs/oasis-core/go/consensus/tendermint/abci"
+	stakingState "github.com/oasislabs/oasis-core/go/consensus/tendermint/apps/staking/state"
+	epochtime "github.com/oasislabs/oasis-core/go/epochtime/api"
+	staking "github.com/oasislabs/oasis-core/go/staking/api"
+)
+
+const (
+	SigningThresholdNumerator   = 3
```

Comment:
These should probably be consensus parameters.

Reply:
aw